apps/aecore/lib/aecore/peers/worker.ex

```diff
@@ -224,6 +224,8 @@ defmodule Aecore.Peers.Worker do
               Logger.info(fn -> "Added #{uri} to the peer list" end)
               Sync.ask_peers_for_unknown_blocks(updated_peers)
               Sync.add_valid_peer_blocks_to_chain()
+              Sync.add_unknown_peer_pool_txs(updated_peers |> Map.values()
```

Comment:
Please move extraction of uri-s to `Sync.add_unknown_peer_pool_txs` as in other functions.